packages/dara-components/js/common/select/select.tsx

```diff
@@ -53,6 +50,11 @@ function hasListSection(items: Item[] | ListSection[]): items is ListSection[] {
 function isStringArray(value: any): value is string[] {
     return Array.isArray(value) && value.every((item) => typeof item === 'string');
 }
+
+/** Type guard to check if an object is an Item */
+function isItem(obj: any): obj is Item {
+    return obj && Object.prototype.hasOwnProperty.call(obj, 'value');
```

Comment:
I've commented on this in the previous thread but it probably got lost in the chain:

`typeof null` resolves to `object` as well (because JavaScript) so you might need to explicitly check that `obj` is not null as well

Reply:
but if obj is null won't the first part of the condition fail?

Reply:
Right sorry I missed your last update 🤦 ignore my comment, this is fine now :)